en/test_and_ci/continous_integration.md

```diff
@@ -1,3 +1,100 @@
 # PX4 Continuous Integration
 
-PX4 builds and testing are performed using GitHub actions and a Jenkins server.
\ No newline at end of file
+PX4 uses continuous integration (CI) to build and run tests against every pull request (PR) and release, and to generate build artifacts such as firmware, documentation updates, and various metadata files needed by QGroundControl.
+
+The integration system uses GitHub actions and a Jenkins server.
+The configuration files for these systems can be found in the [PX4-Autopilot](https://github.com/PX4/PX4-Autopilot) repo main branch:
+- [Jenkinsfile](https://github.com/PX4/PX4-Autopilot/blob/main/Jenkinsfile)
+- [.github/workflows/](https://github.com/PX4/PX4-Autopilot/tree/main/.github/workflows).
+
+The Jenkins build status can be viewed in [ci.px4.io:8080/job/PX4/](http://ci.px4.io:8080/job/PX4/)
```

Comment:
I just discovered that there's 2 Jenkins Server:

- http://ci.px4.io/
- http://px4-jenkins.dagar.ca:8080/blue/organizations/jenkins/PX4-Autopilot/activity

@dagar What is the difference between them? And why do we need two separate Jenkins server?

Reply:
It's just an optimization because Jenkins transfers quite a lot of data between the server and runners and it's also a bit more reliable overall. For the hardware test rack each board is Jenkins slave, but it's just a container all on the same physical system with the Jenkins server.

Reply:
Ah so you mean your Jenkins is the hardware rack testing & px4 Jenkins (CI.px4.io) is for everything else? (Metadata deployment, world magnetic model update, etc)

Reply:
I think more information about test rack and how it runs can be added here. If it is mentioned anywhere in docs a link might suffice